overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -33856,7 +33856,7 @@
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "Note: this quest accepts both LV and MV Distillery.\n\nIf you don\u0027t feel up to making a §3Distillation Tower§r just yet, you can get §9Ethanol§r via a single block §3Distillery§r, a machine that gets one fluid and §cvoids the rest§r (in this case, Water). This defers the tower for a little while longer.\n\nHowever, the Distillation Tower is mandatory for advanced chemistry, as most things have more than one important output: §9Fermented Biomass§r, §9Charcoal Byproducts§r from your §3Pyrolyse Oven§r, and the entire line of petrochemicals from §9Oil§r, to name a few.\n\nCan\u0027t avoid it forever!\n\n",
+          "desc:8": "Note: this quest accepts both LV and MV Distillery.\n\nIf you don\u0027t feel up to making a §3Distillation Tower§r just yet, you can get §9Ethanol§r via a single block §3Distillery§r, a machine that gets one fluid output per §6Integrated Circuit§r configuration, while the §3Distillation Tower§r gives all possible outputs for the input fluid. This defers the tower for a little while longer.\n\nHowever, the Distillation Tower is mandatory for advanced chemistry, as most things have more than one important output: §9Fermented Biomass§r, §9Charcoal Byproducts§r from your §3Pyrolyse Oven§r, and the entire line of petrochemicals from §9Oil§r, to name a few.\n\nCan\u0027t avoid it forever!\n\n",
```

Comment:
I think this change makes it less obvious that they're voiding the other possible distillation outputs by using the Distillery instead of a Tower, and reading it aloud it sounds a bit strangely phrased to me.

We were specifically stating that in the case of Ethanol production, the only other output is just water so they are unlikely to care about it being voided, but generally speaking you should prefer a tower as those other outputs are all useful for something.

Reply:
Well, the other outputs are not voided per se, it is just that you have to use an integrated circuit of a different configuration in the distillery to get the other outputs. So there is no voiding, but you have to select from one of the possible distillation tower outputs.

Reply:
It's not like you can change the circuit configuration to output each fluid one at a time or put in multiple ICs on different settings. If you distill in the single-block machine, you get a single output fluid and the potential other fluids you could have got from using a tower instead are lost forever. In this sense you are voiding the remaining potential outputs, and we want players to realize that's not a good thing the vast majority of the time.

Reply:
That wouldn't make it voiding to me, but rather choosing a specific output from possible outputs. Although the difference is basically semantics. If you can think of a better way to word it, feel free to make the change, or just revert it to the previous wording.

Reply:
Yes, I think it's different ways to conceptualize the same thing: there's a missed opportunity. I believe the prior quest text made it more obvious that this is a serious downside of the machine as compared to the multiblock. The line about IC configurations seems easier to misinterpret, at least to me.

I'll think on it.